home2.1/test_java.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# Specify the binary to test
+# Change this variable based on the type of binary to test
+BINARY_TO_TEST="java task3"
+
+# Function to compare actual result (AR) with expected result (ER)
+compare_results() {
+    AR="$1"
+    ER="$2"
+    if [ "$AR" = "$ER" ]; then
+        echo "PASS: Expected result matches actual result"
+    else
+        echo "FAIL: Expected result does not match actual result"
+    fi
+}
+
+# Test Case 1: New User
+echo "Test Case 1: New User"
```

Comment:
Minor: some preconditions should be set before test execution for example reset history file or put some initial data here cause otherwise test case result will depend on the execution of the other test cases and running a separate case or running in other order may cause issues.